main.star

```diff
@@ -1,37 +1,25 @@
 etcd_module = import_module("github.com/kurtosis-tech/etcd-package/main.star")
 
 KURTOSIS_PACKAGE_INDEXER_IMAGE = "kurtosistech/kurtosis-package-indexer"
-
 KURTOSIS_PACKAGE_INDEXER_DATA_DIR = "/data"
 KURTOSIS_PACKAGE_INDEXER_PORT_ID = "http"
 KURTOSIS_PACKAGE_INDEXER_PORT_NUM = 9770
-
 AWS_S3_BUCKET_SUBFOLDER = "kurtosis-package-indexer"
 
-# Kurtosis Snowflake Account Keys
-KURTOSIS_SNOWFLAKE_ACCOUNT_IDENTIFIER_KEY = "kurtosis_snowflake_account_identifier"
-KURTOSIS_SNOWFLAKE_DB_KEY = "kurtosis_snowflake_db"
-KURTOSIS_SNOWFLAKE_PASSWORD_KEY = "kurtosis_snowflake_password"
-KURTOSIS_SNOWFLAKE_ROLE_KEY = "kurtosis_snowflake_role"
-KURTOSIS_SNOWFLAKE_USER_KEY = "kurtosis_snowflake_user"
-KURTOSIS_SNOWFLAKE_WAREHOUSE_KEY = "kurtosis_snowflake_warehouse"
-
-DEFAULT_LOGGER_LOG_LEVEL = "debug"
-
-
 def run(
     plan,
+    is_running_in_prod="false",
     github_user_token="",
-    kurtosis_package_indexer_version="0.0.7",
+    kurtosis_package_indexer_version="",
     snowflake_env={},
     aws_env={},
-    is_running_in_ci="false",
 ):
     """Runs a Kurtosis package indexer service, listening on port 9770
 
     Args:
-        github_user_token (string): The GitHub user token to use to authenticate to GitHub
-        kurtosis_package_indexer_version (string): The version of the container image to use
+        is_running_in_prod(string): Set to false if devving locally or in CI. (this will ignore any snowflake settings)
```

Comment:
changed `is_running_in_ci` to `is_running_in_prod` - this way for both local + CI, if set to false, metrics reporting is turned off and snowflake env variables don't need to be set.

Reply:
ok, I like it